### PR TITLE
vtkutils for interfacing with VTK functionality

### PR DESCRIPTION
**Pull request by jabooth**
Adds a new submodule to menpo3d, `vtkutils`.

`vtkutils` will be the home of any convenience methods we wish to use from VTK going forwards. For now, two functions and one callable are provided:

1. `trimesh_to_vtk()` function converting a menpo `TriMesh` to a VTK mesh datastructure (`vtkPolyData`)
2. `trimesh_from_vtk()` function converting a VTK mesh to a menpo `TriMesh`
3. `VTKClosestPointLocator` - a callable that can be used to find the nearest point on a 3D surface from a query point.

# Todo
- [x] tests
- [x] guards for checking for 3D meshes in trimesh_to_vtk

### CODE REVIEW

File: menpo3d/test.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import menpo3d
```
Comment (patricksnape) [inline menpo3d/test.py]: ``vtk_test.py`` is a better name?

**Comment by patricksnape** [comment]
Appveyor is failing due to something about expecting int32 rather than int64 - probably need an ``np.require``

File: menpo3d/vtkutils.py [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+    mesh = vtk.vtkPolyData()
+    points = vtk.vtkPoints()
+    points.SetData(numpy_to_vtk(trimesh.points, deep=1))
+    mesh.SetPoints(points)
+
+    cells = vtk.vtkCellArray()
+    cells.SetCells(trimesh.n_tris,
+                   numpy_to_vtkIdTypeArray(
+                       np.hstack((np.ones(trimesh.n_tris)[:, None] * 3,
+                                  trimesh.trilist)).astype(np.int64).ravel(),
```
Comment (patricksnape) [inline menpo3d/vtkutils.py]: This as int64 is failing on Windows - are you sure it is needed? Maybe it should be int32?

Comment (jabooth) [inline menpo3d/vtkutils.py]: @patricksnape I'll change to int32 and let all CI's run, haven't got a Windows box to hand to test with

**Comment by patricksnape** [comment]
+1 with passing tests

**Pull request merged**